package.json

```diff
@@ -10,17 +10,16 @@
   "author": "Openverse <openverse@wordpress.org>",
   "scripts": {
     "postinstall": "pnpm --filter './packages/js/*' run build",
-    "prepare:nuxt": "pnpm --filter frontend run prepare:nuxt",
-    "preeslint": "pnpm --filter '@openverse/eslint-plugin' run build",
-    "eslint": "eslint --ext .js,.ts,.vue,.json,.json5 --ignore-path .gitignore --ignore-path .eslintignore --max-warnings=0 frontend automations/js packages/js .pnpmfile.cjs .eslintrc.js prettier.config.js tsconfig.base.json"
+    "prepare:nuxt": "pnpm --filter frontend run prepare:nuxt"
   },
   "packageManager": "pnpm@9.5.0",
   "engines": {
     "node": ">= 20.0.0 <21"
   },
   "devDependencies": {
+    "@types/node": "22.5.1",
```

Comment:
`@types` shared by multiple libraries should ideally live at the workspace root, to ensure they have the same versions. Otherwise, you can run into nasty errors like `type Plugin is not compatible with argument type Plugin`, where each `Plugin` reference is just referencing a different version of the type by the same name from the same package. This only holds for devDependencies.